src/app/components/Grid/index.jsx

```diff
@@ -46,8 +102,275 @@ export const CPSPageGrid = ({ children, ...props }) => (
+  dir,
+  ...props
+}) => (
+  <Grid
+    {...props}
+    forwardedAs={as}
+    dir={dir}
+    item
+    margins={{
+      group0: true,
+      group1: true,
+      group2: true,
+      group3: true,
+      group4: false,
+      group5: false,
+    }}
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: gridColumnStart,
+    }}
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 5,
+      group4: 5,
+      group5: gridSpan,
+    }}
+  />
+);
+
+export const GridItemConstrainedMediumNoMargin = ({
+  as,
+  dir,
+  gridColumnStart,
+  gridSpan,
+  ...props
+}) => (
+  <Grid
+    {...props}
+    forwardedAs={as}
+    dir={dir}
+    item
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: gridColumnStart,
+    }}
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 5,
+      group4: 5,
+      group5: gridSpan,
+    }}
+  />
+);
+
+export const GridItemConstrainedLarge = ({ as, dir, ...props }) => (
+  <Grid
+    {...props}
+    dir={dir}
+    forwardedAs={as}
+    item
+    margins={{
+      group0: true,
+      group1: true,
+      group2: true,
+      group3: true,
+      group4: false,
+      group5: false,
+    }}
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: 5,
+    }}
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 6,
+      group4: 6,
+      group5: 12,
+    }}
+  />
+);
+
+export const GridItemConstrainedLargeNoMargin = ({ as, dir, ...props }) => (
+  <Grid
+    {...props}
+    forwardedAs={as}
+    dir={dir}
+    item
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: 5,
+    }}
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 6,
+      group4: 6,
+      group5: 12,
+    }}
+  />
+);
+
+// 1.
+// The max-height must be 0 at Group 5 breakpoints so that
+// the item does not force the following sibling item downwards.
+
+const PopOutAtGroup5 = styled(GridItemConstrainedMedium)`
+  @supports (display: grid) {
+    @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+      max-height: 0; /* [1] */
+      padding-top: 0.25rem;
+    }
+  }
+`;
+export const PopOutGridItemMedium = props => {
+  const { children } = props;
+  return <PopOutAtGroup5 {...props}>{children}</PopOutAtGroup5>;
+};
+
+ArticlePageGrid.propTypes = {
+  // eslint-disable-next-line react/require-default-props
+  as: string,
+};
+
+GridItemConstrainedLarge.propTypes = {
+  // eslint-disable-next-line react/require-default-props
+  as: string,
+  // eslint-disable-next-line react/require-default-props
+  dir: string,
+};
+
+GridItemConstrainedLargeNoMargin.propTypes = {
+  // eslint-disable-next-line react/require-default-props
+  as: string,
```

Comment:
to remove the `// eslint-disable-next-line react/require-default-props` I think the default as prop could be `undefined`. The dir could be `ltr`